jscripts/myalerts.js

```diff
@@ -4,22 +4,35 @@
 
     this.MybbStuff.MyAlerts = (function MyAlertsModule(window, $) {
         var module = function MyAlerts() {
-            var unreadAlertsProxy = $.proxy(this.getUnreadAlerts, this),
+            var latestAlertsProxy = $.proxy(this.getLatestAlerts, this),
                 deleteAlertProxy = $.proxy(this.deleteAlert, this),
                 markAllReadProxy = $.proxy(this.markAllRead, this),
                 markReadAlertProxy = $.proxy(this.markReadAlert, this),
                 bodySelector = $("body");
 
-            bodySelector.on("click", "#getUnreadAlerts", unreadAlertsProxy);
+            var urlGetLatest = (typeof myAlertsBcMode !== 'undefined' && myAlertsBcMode == '1')
```

Comment:
Might just be me, but I can't see where this gets set in this merge request?

Reply:
You're right, it's not set in this merge request. It was set in #299 which I merged myself after flagging I'd do so after about a week in the absence of objections. You can still object to it though!